vignettes/examples/multiple_linear_multilogit.Rmd

```diff
@@ -0,0 +1,31 @@
+### Multiple multinomial regression
+
+A multi-variable Bayesian linear regression model using the iris data.
+
+```{r multiple_linear_multilogit_greta}
+data(iris)
+
+library(tensorflow)
+op <- greta::.internals$nodes$constructors$op
```

Comment:
there's `imultilogit()` now, which takes one fewer columns that the number of response columns, since one is the baseline/constrast